pallets/sto/src/benchmarking.rs

```diff
@@ -0,0 +1,215 @@
+#![cfg(feature = "runtime-benchmarks")]
+use crate::*;
+use core::convert::TryFrom;
+use frame_benchmarking::benchmarks;
+use frame_support::traits::Get;
+use frame_system::RawOrigin;
+use pallet_settlement::{benchmarking::compliance_setup, VenueDetails};
+use polymesh_common_utilities::benchs::User;
+use polymesh_common_utilities::{asset::AssetType, benchs::UserBuilder};
+use polymesh_primitives::TrustedIssuer;
+
+pub type Asset<T> = pallet_asset::Module<T>;
+pub type ComplianceManager<T> = pallet_compliance_manager::Module<T>;
+pub type Identity<T> = identity::Module<T>;
+pub type Timestamp<T> = pallet_timestamp::Module<T>;
+pub type Settlement<T> = pallet_settlement::Module<T>;
+pub type Sto<T> = crate::Module<T>;
+
+fn create_asset<T: Trait>(
+    origin: RawOrigin<T::AccountId>,
+    ticker: Ticker,
+    supply: u128,
+) -> DispatchResult {
+    <Asset<T>>::create_asset(
+        origin.into(),
+        vec![b'A'].into(),
+        ticker,
+        supply.into(),
+        true,
+        AssetType::default(),
+        vec![],
+        None,
+    )
+}
+
+fn add_compliance<T: Trait>(origin: RawOrigin<T::AccountId>, ticker: Ticker) -> DispatchResult {
+    //TODO: replace with worst case rules
+    <ComplianceManager<T>>::add_compliance_requirement(origin.into(), ticker, vec![], vec![])
+}
```

Comment:
Why not use `compliance_setup` here?

Reply:
This is used when weight does not depend on having complex rules so it just adds empty rules. I should rename it so that's more obvious.